File: terceiroSemestre/poo/polimorfismo/java/src/ContatoFisico.java
public class ContatoFisico() extends Contato{
    private String dataNascimento;
    private String estadoCivil;
    private String cpf;
    
    ContatoFisico(String nome, String endereço, String email, 
        String dataNascimento, String estadoCivil, String cpf){
        super(nome, endereço, email);
        this.dataNascimento = dataNascimento;
        this.estadoCivil = estadoCivil;
        this.cpf = cpf;
    }

    String toString(){
        super.toString();
        System.out.println("Data de nascimento: " + dataNascimento);
        System.out.println("Estado civil: " + estadoCivil);
        System.out.println("CPF: " + cpf);
    }
}

File: 3rdPeriod/ObjectOrientedProgramming/polimorfismo/python/ex2/contato_fisico.py
import contato

class ContatoFisico(contato.Contato):
    def __init__(self, nome, endereço, email, nascimento, estadoCivil, cpf):
        super().__init__(nome, endereço, email)
        self.nascimento = nascimento
        self.estadoCivil = estadoCivil
        self.cpf = cpf

    def __str__(self):
        super().__str__()
        print("Data de Nascimento = " + self.nascimento)
        print("Estado Civil: " + self.estadoCivil)
        print("CPF: " + self.cpf)

        